vm/src/bytecode.rs

```diff
@@ -50,9 +56,11 @@ pub enum Instruction {
     },
     LoadName {
         name: String,
+        scope: NameScope,
     },
     StoreName {
         name: String,
+        scope: NameScope,
     },
     DeleteName {
```

Comment:
```DeleteName``` needs a scope too?

Reply:
Yes, it does. Do you want me to implement this now?

Reply:
Now or later is fine.